app/main.py

```diff
@@ -2,24 +1,101 @@
+from __future__ import annotations
+from abc import ABC, abstractmethod
 
 
 class IntegerRange:
-    pass
+    def __init__(
+            self,
+            min_amount: int,
+            max_amount: int
+    ) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(
+            self,
+            owner: SlideLimitationValidator,
+            name: str
+    ) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(
+            self: IntegerRange,
+            instance: SlideLimitationValidator,
+            owner: SlideLimitationValidator
+    ) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(
+            self: IntegerRange,
+            instance: SlideLimitationValidator,
+            value: int
+    ) -> None:
+        try:
+            if not self.min_amount <= value <= self.max_amount:
+                raise ValueError
+        except ValueError:
+            instance.result = False
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    @abstractmethod
+    def __init__(
+            self,
+            age: int,
+            weight: int,
+            height: int
+    ) -> None:
+        self.result = True
+        self.age = age
+        self.weight = weight
+        self.height = height
+
+    def checking(self) -> bool:
```

Comment:
The `checking` method is a bit unclear. The name of the method should be a verb or a verb phrase and should indicate what the method does. Consider renaming this method to something more descriptive, like `is_valid` or `check_validity`.